compilers/kfp-sdk/README.md

```diff
@@ -0,0 +1,41 @@
+### KFP SDK Compiler
+
+KFP SDK Compiler is a tool that compiles a KFP SDK pipeline definition into Kubeflow Pipeline representation.
```

Comment:
I assume you want to say 'a representation that can be understood by Kubeflow Pipelines`
```suggestion
KFP SDK Compiler is a tool that compiles a KFP SDK pipeline definition into a Kubeflow Pipelines representation.
```